docs/reference/migration/migrate_8_0/scripting.asciidoc

```diff
@@ -22,4 +22,29 @@ scripts. Any use of `getDayOfWeek` expecting a return value of `int` will result
 in a compilation error or runtime error and may not allow the upgraded node to
 start.
 ====
+
+.Empty scripts/templates are no longer allowed as stored scripts.
+[%collapsible]
+====
+*Details* +
+Empty scripts/templates cannot be specified as the source of a
+PutStoredScript action.
```

Comment:
Is there a reason we reference the `PutStoredScript` action rather than the [create or update stored script API](https://www.elastic.co/guide/en/elasticsearch/reference/current/create-stored-script-api.html) here?

It doesn't look like it in the code, but the wording made me wonder if this only impacted the Java client.

Reply:
No! Just a mistake on my part.